browser-test/bin/truncate_tables.sh

```diff
@@ -2,7 +2,15 @@
 
 set -euo pipefail
 
-cat <<EOF | PGPASSWORD=example psql -h db -p 5432 -U postgres 
+if [[ ${BASE_URL-not_set} == "http://localhost:9999" ]]; then
+    host=localhost
+    port=2345
+else
+    host=db
+    port=5432
+fi
+
+cat <<EOF | PGPASSWORD=example psql -h $host -p $port -U postgres
```

Comment:
Thoughts (non-blocking): any chance we can run this in a docker container? `postgres` image has this CLI. My environment doesn't have psql installed that's why I asked 😛 .

Reply:
Sure, I added some code to do this but it's a little tricky to test since I do have `psql` locally. Mind pulling it down and running `./bin/run-browser-tests-local` to see if it works for you?

Reply:
It is failing saying some command/file is not found

```
% ./bin/run-browser-tests-local 
yarn run v1.22.10
$ ./bin/truncate_tables.sh && jest --forceExit --maxConcurrency=1
[FATAL tini (7)] exec -e failed: No such file or directory
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
Pushed a fix to branch. I can confirm this works for my setup.

Reply:
Thanks for making the change!

Reply:
Oh great, thank you!